india_compliance/gst_india/constants/custom_fields.py

```diff
@@ -861,6 +876,28 @@
     "Sales Invoice": reverse_charge_field.copy().update(insert_after="is_debit_note"),
 }
 
+exclude_from_gst_field = frappe._dict(
+    fieldname="exclude_from_gst",
+    label="Exclude from GST Reporting",
+    fieldtype="Check",
+    print_hide=1,
+    default=0,
+    insert_after="company",
+)
+
+EXCLUDE_FROM_GST_FIELDS = {
+    "Quotation": exclude_from_gst_field,
```

Comment:
- sales: is_export or company
- pos invoice: is_pos
- purchase: Is Reverse Charge